linode/manager
# M3 Add find related tests to Jest config

## Description

Update our pre-commit hook to only run tests related to changed files.

## To Test

- Change a component in such a way as to break a test
- Try to commit the change
- Commit should fail, but only a few test files should run.
